contracts/multisig/src/contract/execute.rs

```diff
@@ -211,19 +214,25 @@ pub fn authorize_callers(
 
 pub fn unauthorize_callers(
     deps: DepsMut,
-    contracts: HashMap<Addr, ChainName>,
+    contracts: Vec<Addr>,
```

Comment:
```suggestion
    callers: Vec<Addr>,
```
probably better if we use `callers` to match the function name?